### PR TITLE
Issue 22041 activities for systemtags

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Activities for admins when tags are managed:
  - admin created system tag `Test`
  - admin updated system tag `Test` to `Test 1 (not-assignable)`
  - admin updated system tag `Test 1 (not-assignable)` to `Test 3 (invisible)`
  - admin deleted system tag `Test 3 (invisible)`
- [x] Activities for users with access to a file, when a tag is (un-)assigned via the web UI:
  - admin assigned system tag `Test` to `folder_test`
  - admin unassigned system tag `Test 3 (invisible)` from `folder_test`

### CODE REVIEW

**Comment by karlitschek** [comment]
sweet :+1:

**Comment by nickvergessen** [comment]
Failure is unrelated:
```
09:58:11 + git clone -b master https://github.com/owncloud/smashbox.git
09:58:11 Cloning into 'smashbox'...
09:59:29 fatal: unable to access 'https://github.com/owncloud/smashbox.git/': Empty reply from server
```

**Comment by nickvergessen** [comment]
Ready for review @PVince81 @rullzer

File: apps/systemtags/activity/extension.php [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+	 * @return string|false
+	 */
+	public function translate($app, $text, $params, $stripPath, $highlightParams, $languageCode) {
+		if ($app !== self::APP_NAME) {
+			return false;
+		}
+
+		$l = $this->getL10N($languageCode);
+
+		/*
```
Comment (PVince81) [inline apps/systemtags/activity/extension.php]: remove ?

else add a "TODO: uncomment when XYZ"

File: apps/systemtags/activity/extension.php [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+				case self::DELETE_TAG:
+					return array(
+						0 => 'username',
+						//1 => 'systemtag description',
+					);
+				case self::UPDATE_TAG:
+					return array(
+						0 => 'username',
+						//1 => 'systemtag description',
+						//2 => 'systemtag description',
```
Comment (PVince81) [inline apps/systemtags/activity/extension.php]: what about these ? for information purposes?

File: apps/systemtags/activity/extension.php (hunk 1 above)
Comment (nickvergessen) [inline apps/systemtags/activity/extension.php]: Right, forgot to fix this

File: apps/systemtags/appinfo/info.xml [hunk 3]
```diff
@@ -6,11 +6,14 @@
 	<licence>AGPL</licence>
 	<author>Vincent Petry</author>
 	<default_enable/>
-	<version>0.1</version>
+	<version>0.2</version>
```
Comment (PVince81) [inline apps/systemtags/appinfo/info.xml]: feel free to add your name as author :wink:

File: apps/systemtags/activity/extension.php (hunk 2 above)
Comment (nickvergessen) [inline apps/systemtags/activity/extension.php]: yes

**Comment by PVince81** [comment]
Tested, works :+1:

**Comment by rullzer** [comment]
Whoohoo more activities!
Tested a few tags and worked.
Code looks good.
 :+1:

**Pull request merged**